src/mscorlib/src/System/BitConverter.cs

```diff
@@ -495,5 +452,20 @@ public static unsafe float Int32BitsToSingle(int value)
         {
             return *((float*)&value);
         }
+
+        private static void ThrowValueArgumentNull()
+        {
+            throw new ArgumentNullException("value");
```

Comment:
Not really ... just copied what was in corefx. I have fixed it up to use ThrowHelper.